### PR TITLE
Sync with latest pypi mobans

### PR DESCRIPTION
**Pull request by chfw**
it would show up a lot of badges, at least.

### CODE REVIEW

**Comment by chfw** [comment]
replaced old mobanfile syntax 

```
requires:
   - pypi-mobans-pkg==0.0.2
 configuration:
   template_dir:
   - "pypi-mobans-pkg:templates"

```

with 

 ```
configuration:
   template_dir
   - "git://github.com/moremoban/pypi-mobans?submodule=true&branch=dev!/templates"
   - "git://github.com/moremoban/pypi-mobans?submodule=true&branch=dev!/statics"
```

**Comment by chfw** [comment]
@CLiu13 , by the way, do you have pypi user name? I will add your user name to this repo and moban-handlebars.

**Comment by CLiu13** [comment]
> @CLiu13 , by the way, do you have pypi user name? I will add your user name to this repo and moban-handlebars.

Yep, just created an account - username is same as my GitHub account, ```CLiu13```

File: setup.py [hunk 1]
```diff
@@ -1,61 +1,81 @@
...
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+
 ]
 
+
 INSTALL_REQUIRES = [
-    'slimish-jinja>=1.1.0',
-    'lml>=0.0.7',
-    'jinja2-fsloader>=0.2.0',
```
Comment (CLiu13) [inline setup.py:44]: ```jinja2-fsloader``` is needed, can't remove it AFAIK

**Review by CLiu13 — APPROVED** [review]
LGTM 👍

**Comment by chfw** [comment]
nice team work!

**Pull request merged**